### PR TITLE
Update docs and images with removal of notary

### PR DESCRIPTION
**Pull request by alexru26**
1. Fixes https://github.com/goharbor/harbor/issues/18631

### CODE REVIEW

**Comment by OrlinVasilev** [comment]
@alexru26 thank you for that PR it's great :))

**Comment by OrlinVasilev** [comment]
@AllForNothing can you please review

**Comment by AllForNothing** [comment]
@OrlinVasilev Sure

**Comment by OrlinVasilev** [comment]
thank you!

File: docs/build-customize-contribute/swagger.yaml [hunk 1]
```diff
@@ -1200,7 +1200,7 @@ paths:
     get:
       summary: Get the tag of the repository.
       description: |
-        This endpoint aims to retrieve the tag of the repository. If deployed with Notary, the signature property of response represents whether the image is singed or not. If the property is null, the image is unsigned.
+        This endpoint aims to retrieve the tag of the repository. If the property is null, the image is unsigned.
```
Comment (AllForNothing) [inline docs/build-customize-contribute/swagger.yaml:1203]: change it to `This endpoint aims to retrieve the tag of the repository.`

File: docs/build-customize-contribute/swagger.yaml [hunk 2]
```diff
@@ -1253,7 +1253,7 @@ paths:
     get:
       summary: Get tags of a relevant repository.
       description: |
-        This endpoint aims to retrieve tags from a relevant repository. If deployed with Notary, the signature property of response represents whether the image is singed or not. If the property is null, the image is unsigned.
+        This endpoint aims to retrieve tags from a relevant repository. If the property is null, the image is unsigned.
```
Comment (AllForNothing) [inline docs/build-customize-contribute/swagger.yaml:1256]: change it to `This endpoint aims to retrieve tags from a relevant repository.`

**Review by AllForNothing — APPROVED** [review]
lgtm

**Review by MinerYang — APPROVED** [review]
lgtm

**Pull request merged**